scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1408,43 +1566,43 @@ def addDBVersion():
     isd_admin_password = sys.argv[16]
     sapor_host_url = sys.argv[17]
     audit_service_url = sys.argv[18]
-    redis_host = sys.argv[19]
-    redis_port = sys.argv[20]
-    redis_password = sys.argv[21]
+    spin_db_type = sys.argv[19]        
+    redis_host = sys.argv[20]
```

Comment:
@sudhakaropsmx we should conditonally read redis or sql params not both. Means, initialize redis variables if spin_db_type is redis otherwise initialize sql related variables (not both at a time)

Reply:
Updated as suggested